### PR TITLE
Makes version selector sticky

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Makes version selector sticky

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _sass/custom/custom.scss [hunk 1]
```diff
@@ -48,11 +48,12 @@ code {
 }
 
 .site-nav {
-  padding-top: 1rem;
+  padding-top: 0;
   position: sticky;
-  top: 1rem;
+  top: 0;
   max-height: calc(100vh - 2rem);
```
Comment (AMoo-Miki) [inline _sass/custom/custom.scss:54]: Do we need `- 2rem` now that the `top` and `padding-top` are `0`?

Comment (kolchfa-aws) [reply _sass/custom/custom.scss]: We don't, thank you for catching this. Updated.

**Review by ariamarble — APPROVED** [review]

**Review by hdhalter — APPROVED** [review]
This is great, thanks, Fanit!

**Review by carolxob — APPROVED** [review]
LGTM!

**Pull request merged**